_commands/tools/scp.md

```diff
@@ -3,22 +3,35 @@
 
 scp
 -------
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+`scp`, or _Secure Copy_, is used to securely transfer files based on the _Secure Shell protocol_ (`ssh`). Unlike `cp`, at least one of the source or destination should be a remote host.
```

Comment:
mebe instead of "at least one of the source or destination should be a remote host", something more like "`scp` allows files to be transferred from or to a remote host"?